setup.py

```diff
@@ -69,7 +69,7 @@
           "Topic :: Utilities",
       ],
       package_data={
-          'func_adl_xAOD': ['R21Code/*'],
+          'func_adl_xAOD': ['template/atlas/r21/*'],
```

Comment:
Ok - the cms templates need to be added here! Not as another variable. Sorry - dead code above. But here is important.

Reply:
Done